bindgen-integration/src/lib.rs

```diff
@@ -282,3 +282,9 @@ fn test_virtual_dtor() {
         assert_eq!(bindings::VirtualDestructor_sDestructorCount, 1);
     }
 }
+
+#[test]
+fn test_item_rename() {
+    assert_eq!(bindings::CONST_VALUE, 3);
+    assert_eq!(bindings::function_name(), 4);
```

Comment:
This doesn't build (need `unsafe { }` around the function call).

Reply:
My bad I ran the tests in the root folder instead of bindgen-integration before pushing.